crates/wasm-encoder/src/instances.rs

```diff
@@ -1,28 +1,30 @@
-use super::*;
+use crate::{encoders, ComponentSection, ComponentSectionId, Export};
```

Comment:
FWIW this file is sort of what I would naively expect to be the shape of most of the module linking transition to the new component model format. Most of the previous module linking implementation is still applicable, everything just has a slightly different shape with a few updates here and there.